app/request_lookup.py

```diff
@@ -25,19 +47,31 @@ def get_results(register, data_type, search_value):
 
     result_output = []
     results = json.loads(resp.text)
+
+    logger.info('Query Successful', took=results.get('took'))
+
     for result in results['hits']['hits']:
-        occupation = result['highlight'][data_type][0] if 'highlight' in result else result['_source'][data_type]
-        result_output.append(occupation)
+        result_value = {
+            'en': result['_source'].get('en')
```

Comment:
Don't quite get why this is `en` and not `language` var?

Reply:
it always returns the english value